test/manager.js

```diff
@@ -1091,6 +1091,56 @@ describe('Manager', () => {
             expect(rendered).to.contain('<h1>default message</h1>');
         });
 
+        it('renders with an async global context function (no request)', async () => {
```

Comment:
Can we use the test with this description as a template for these tests instead? `renders with a global context object` We don't need to test that we can render context without a request again

Mainly noting that we don't need the part on the `message` context property that says
`request ? request.route.path : ...`, we can just use plain values

Reply:
Sure thing. I figured it was copy pasta from the previous tests. I'll fix it when I get a chance.